x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -346,6 +347,10 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
             type: 'data_backfill',
             backfillFn: migratePackagePolicyToV81102,
           },
+          {
```

Comment:
I believe we need to create a new version. number 4

Reply:
seems like when I add a new version (number 4) it breaks the integration test, I might be missing something on the version 4 but not sure what 👀

Reply:
Which test is failing?

Reply:
@juliaElastic  Jest Integration Tests #4 / checking migration metadata changes on all registered SO types detecting migration related changes in registered types 
as shown in the build https://buildkite.com/elastic/kibana-pull-request/builds/164566

Reply:
hm not sure about that, maybe you can ask @elastic/kibana-core

Reply:
when adding migrations, running `node scripts/jest_integration -u src/core/server/integration_tests/saved_objects/migrations/group2/check_registered_types.test.ts` fixes the migration hash of the SO